Source/SteveSloth/Private/MyEnemy.cpp

```diff
@@ -0,0 +1,20 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+
+#include "MyEnemy.h"
+
+MyEnemy::MyEnemy(float MaxHealth, float CurrentHealth, int Damage, enum MovementState, enum AttackType, enum Level, float MoveSpeed, bool IsDead)
```

Comment:
This is considered an Overloaded Class Function
Curious as to why you would do this. This would mean the only way to access info is if we enter every variable in the call. This is not how this should function.
Please remove the variables and just have MyEnemy::MyEnemy()

then inside MyEnemy

this is where you would declare your variables.